modules/ingestor/src/service/advisory/csaf/loader.rs

```diff
@@ -98,51 +98,54 @@ impl<'g> CsafLoader<'g> {
         report: &dyn ReportSink,
         tx: TX,
     ) -> Result<(), Error> {
-        if let Some(cve_id) = &vulnerability.cve {
-            self.graph.ingest_vulnerability(cve_id, (), &tx).await?;
-
-            let advisory_vulnerability = advisory
-                .link_to_vulnerability(
-                    cve_id,
-                    Some(AdvisoryVulnerabilityInformation {
-                        title: vulnerability.title.clone(),
-                        summary: None,
-                        description: None,
-                        discovery_date: vulnerability.discovery_date.and_then(|date| {
-                            OffsetDateTime::from_unix_timestamp(date.timestamp()).ok()
-                        }),
-                        release_date: vulnerability.release_date.and_then(|date| {
-                            OffsetDateTime::from_unix_timestamp(date.timestamp()).ok()
-                        }),
-                        cwes: vulnerability.cwe.as_ref().map(|cwe| vec![cwe.id.clone()]),
+        let Some(cve_id) = &vulnerability.cve else {
```

Comment:
I need to get `let/else` into my brain. Nice.

Reply:
It can safe you one level of indentation. I like that.